hash/thinkpad8xx.xml

```diff
@@ -0,0 +1,88 @@
+		<description>AIX 4.1.5</description>
+		<year>1997</year>
+		<publisher>IBM</publisher>
+		<sharedfeat name="compatibility" value="800,820,850" />
+		<part name="cdrom1" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="aix_4.1.5_disc_01" sha1="4c49c8190e33c2c7f13a74d7332d384edd6b3042" />
+			</diskarea>
+		</part>
+		<part name="cdrom2" interface="cdrom">
+			<diskarea name="cdrom">
+				<disk name="aix_4.1.5_disc_01" sha1="228afedd90619e89c6b466b4970bda0e5a3e9a13" />
+			</diskarea>
+		</part>
+	</software>
+
+	<!-- Only supported on Thinkpad Series 820 and 850 -->
+	<software name="os2w_ppc">
+		<description>OS/2 Warp PowerPC Edition</description>
+		<year>1995</year>
+		<publisher>IBM</publisher>
+		<sharedfeat name="compatibility" value="820,850" />
+		<part name="cdrom1" interface="cdrom">
+			<feature name="part_number" value="19H1133"/>
+			<diskarea name="cdrom">
+				<disk name="aix_4.1.5_disc_01" sha1="b646fdb3ab5f982304e7cbaf6bd735e20cc66198" />
+			</diskarea>
+		</part>
+		<part name="cdrom2" interface="cdrom">
+			<feature name="part_number" value="19H1134"/>
+			<diskarea name="cdrom">
+				<!-- Applications Sampler December, 1995-->
+				<disk name="aix_4.1.5_disc_01" sha1="e1fd3631c353b2903d35658a59e8a79b6306c1d2" />
+			</diskarea>
+		</part>
+	</software>
+
+	<software name="sol_2_5_1_ppc">
+		<description>Solaris 2.5.1 PowerPC Platform Edition</description>
+		<year>1996</year>
+		<publisher>Sun Microsystem</publisher>
+		<sharedfeat name="compatibility" value="800,820,850" />
+		<part name="flop" interface="floppy">
+			<feature name="part_number" value="702-4348-06"/>
+			<dataarea name="flop" size="168000">
+				<!-- Solaris MDB Boot 2.5.1 Release PowerPC Platform Edition (IBM Hardware Platform) -->
+				<rom name="solaris_2_5_1_mdb_boot_ppc_ibm.img" size="168000" crc="b45832e2" sha1="be9e11364816ab5f3362396ed0f457d4a2e61188" />
+			</dataarea>
+		</part>
+		<part name="cdrom1" interface="cdrom">
+			<feature name="part_number" value="704-5237-06"/>
+			<diskarea name="cdrom">
+				<disk name="solaris_2.5.1_powerpc" sha1="61f003bf259e472673b6f65b18ed9b2629ea396b" />
+			</diskarea>
+		</part>
+		<part name="cdrom2" interface="cdrom">
+			<feature name="part_number" value="704-5507-06"/>
+			<diskarea name="cdrom">
+				<!-- Solaris Desktop 1.1 PowerPC Platform Edition -->
+				<disk name="solaris_desktop_1.1_powerpc" sha1="00502ae94b626f0fc3a080ba1c419397b8aaaae3" />
+			</diskarea>
+		</part>
+	</software>
+
+	<software name="win_nt_3_5_ppc">
+		<description>Windows NT Workstation 3.5 PowerPC Edition</description>
+		<year>1994</year>
+		<publisher>IBM</publisher>
+		<sharedfeat name="compatibility" value="800,820,850" />
+		<part name="cdrom" interface="cdrom">
+		<feature name="part_number" value="11H5926"/>
+			<diskarea name="cdrom">
+				<disk name="windows_nt_3.5_workstation_powerpc" sha1="ea1035d1c7f68d8237926125e9a9ccae83fa3f82" />
+			</diskarea>
+		</part>
+	</software>
```

Comment:
Should this come with a floppy for installing the ARC second stage boot loader on the hard disk?

Reply:
Yes!!!!!!
But I've found several version of the floppy images: v1.46, v1.48, v1.49, v1.50 and v1.51. 
I've added only the v1.51. How should I add all the others? 
Creating new entries? Something like "Windows NT Workstation 3.5 PowerPC Edition with ARC Boot Firmware v1.51", "Windows NT Workstation 3.5 PowerPC Edition with ARC Boot Firmware v1.50", etc.?
Adding all floppy versions to the same entry?

Reply:
Not sure what to do about them at the moment.  I guess this raises the issue that this release of WinNT supported PReP machines in general, and not just these ThinkPads, but we don’t have a software list for PReP software yet (which isn’t a surprise given we don’t support required PowerPC CPU features).